raiden/tests/integration/long_running/test_settlement.py

```diff
@@ -76,33 +78,30 @@ def run_test_settle_is_automatically_called(raiden_network, token_addresses):
 
     # A ChannelClose event will be generated, this will be polled by both apps
     # and each must start a task for calling settle
-    RaidenAPI(app1.raiden).channel_close(registry_address, token_address, app0.raiden.address,
-                                         app0.raiden.alarm.sleep_time)
+    RaidenAPI(app1.raiden).channel_close(registry_address, token_address, app0.raiden.address)
 
     waiting.wait_for_close(
         app0.raiden,
         registry_address,
         token_address,
         [channel_identifier],
         app0.raiden.alarm.sleep_time,
-        app1.raiden.address
+        app0.raiden.address
     )
 
-    channel_state = views.get_channelstate_for_close_channel(
-        views.state_from_raiden(app0.raiden), registry_address, token_address, app0.raiden.address, app1.raiden.address,
-        channel_identifier
+    channel_state = views.get_channelstate_for(
+        views.state_from_raiden(app0.raiden), registry_address, token_address, app1.raiden.address
```

Comment:
Can we re'review this? This function needs more arguments: 
```
def get_channelstate_for_close_channel(
    chain_state: ChainState,
    payment_network_id: PaymentNetworkID,
    token_address: TokenAddress,
    creator_address: Address,
    partner_address: Address,
    channel_id_to_check: ChannelID = None
) -> Optional[NettingChannelState]:
```

Reply:
This function is get_channel_state_for, the only missing param is the partner address: 

```
def get_channelstate_for(
    chain_state: ChainState,
    payment_network_id: PaymentNetworkID,
    token_address: TokenAddress,
    creator_address: Address = None,
    partner_address: Address = None
) -> Optional[NettingChannelState]:
```

and in this case is not needed, since the app1 address is the creator of the channel.

Reply:
Ups! You're right! Mb here!